tests/basic/basic_test.go

```diff
@@ -194,6 +194,46 @@ var _ = Describe("AppTasksDown", func() {
 	})
 })
 
+// This test scales up and down an application and checks if app has actually scaled accordingly
+var _ = Describe("AppScaleUpAndDown", func() {
+	It("has to scale up and scale down the app", func() {
+		var err error
+		scaleUpFactor := int32(3)
```

Comment:
let's not hardcode the scale factor. Add interface in scheduler to get current scale factor and the test should do +1 and -1 on that.